Framework/CurveFitting/src/Functions/CrystalFieldMultiSpectrum.cpp

```diff
@@ -276,9 +275,8 @@ void CrystalFieldMultiSpectrum::buildTargetFunction() const {
     }
   } else {
     m_physprops.clear();
-    for (auto elem : physprops) {
-      m_physprops.emplace_back(static_cast<int>(elem));
-    }
+    std::transform(physprops.begin(), physprops.end(), std::back_inserter(m_physprops),
```

Comment:
Nice this is looking much better with the std::transform! This is a very picky thing but can this use constant iterators `cbegin` and `cend`? And is it possible to reserve memory in `m_physprops` based on the size of `physprops` before doing the transform?

Reply:
seems like a good idea - didn't know about the constant iterators!